tests/test_normalize_by_median.py

```diff
@@ -336,10 +337,25 @@ def test_normalize_by_median_double_file_name():
 
     try:
         (status, out, err) = utils.runscript(script, args, in_dir)
+        raise Exception("Shouldn't get to this")
```

Comment:
Right, that makes me un-happy too :-( Why is runscript now throwing AssertionErrors?